### PR TITLE
Remove artifacts from `dev`

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Same as dash-table https://github.com/plotly/dash-table/pull/603

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,240 +1,240 @@
 version: 2
 jobs:
-  "percy-finalize":
-    docker:
-      - image: percyio/agent
-    steps:
-      - run: percy finalize --all
+    'percy-finalize':
```
Comment (byronz) [inline .circleci/config.yml]: does this file just get `prettier`? hard to tell from the diff the real change

Comment (Marc-Andre-Rivet) [reply .circleci/config.yml]: fixed

File: .circleci/config.yml [hunk 2]
```diff
@@ -77,10 +77,10 @@ jobs:
       - run:
           name: 🏗️ build dash
           command: |
-            . venv/bin/activate && pip install --upgrade -e . --quiet && mkdir packages
+            . venv/bin/activate && mkdir packages
             # build main dash
             git clone --depth 1 https://github.com/plotly/dash.git dash-main
-            cd dash-main && pip install -e .[dev,testing] && python setup.py sdist && mv dist/* ../packages/
+            cd dash-main && pip install -e .[dev,testing] --quiet && python setup.py sdist && mv dist/* ../packages/
```
Comment (byronz) [inline .circleci/config.yml]: I think our intention is to mute the progress bar,  ` --progress-bar off`
at least for now, we need to check the pip install logs from time to time

Comment (Marc-Andre-Rivet) [reply .circleci/config.yml]: https://github.com/plotly/dash-core-components/pull/659/commits/56093b14ebd5ca7170da85b61fa3cf781e686c8c

**Pull request merged**